### ISSUE TITLE
Building CGAL on CI server

### ISSUE THREAD

**Issue by krober10nd** [CGAL/cgal#5390]
Hello, 

Recently on a CI server where I unit test a code of mine, a docker with an older version of Python (3.6.5) stopped "building" (I realize it's a header-only library now) the latest head CGAL during the installation step. 

I receive the following error from CMake (3.7.2-1)

Has anything recently changed to prompt this error? I've been testing against this identical docker for the last 4 months without fail until 4 days ago.

```
 running build_ext
  -- The C compiler identification is GNU 6.3.0
  -- The CXX compiler identification is GNU 6.3.0
  -- Check for working C compiler: /usr/bin/cc
  -- Check for working C compiler: /usr/bin/cc -- works
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Check for working CXX compiler: /usr/bin/c++
  -- Check for working CXX compiler: /usr/bin/c++ -- works
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- OS detected: Linux
  -- CXX Compiler detected: GNU
  -- CMake additional search path for libraries:
  -- Using header-only CGAL
  -- Targetting Unix Makefiles
  -- Using /usr/bin/c++ compiler.
  -- Found GMP: /usr/lib/x86_64-linux-gnu/libgmp.so
  -- Found MPFR: /usr/lib/x86_64-linux-gnu/libmpfr.so
  -- Boost version: 1.62.0
  -- Boost include dirs: /usr/include
  -- Boost libraries:
  CMake Error at /usr/local/lib/cmake/CGAL/CGALConfig.cmake:135 (add_library):
    add_library cannot create ALIAS target "CGAL::CGAL" because target "CGAL"
    is IMPORTED.
  Call Stack (most recent call first):
    CMakeLists.txt:16 (find_package)
  
  
  CMake Error at /usr/local/lib/cmake/CGAL/CGAL_SetupGMP.cmake:55 (target_include_directories):
    Cannot specify include directories for imported target "CGAL".
  Call Stack (most recent call first):
    /usr/local/lib/cmake/CGAL/CGAL_SetupCGALDependencies.cmake:80 (use_CGAL_GMP_support)
    /usr/local/lib/cmake/CGAL/CGAL_setup_target_dependencies.cmake:3 (CGAL_setup_CGAL_dependencies)
    /usr/local/lib/cmake/CGAL/CGALConfig.cmake:137 (CGAL_setup_target_dependencies)
    CMakeLists.txt:16 (find_package)
  
  
  CMake Error at /usr/local/lib/cmake/CGAL/CGAL_SetupGMP.cmake:62 (target_include_directories):
    Cannot specify include directories for imported target "CGAL".
  Call Stack (most recent call first):
    /usr/local/lib/cmake/CGAL/CGAL_SetupCGALDependencies.cmake:80 (use_CGAL_GMP_support)
    /usr/local/lib/cmake/CGAL/CGAL_setup_target_dependencies.cmake:3 (CGAL_setup_CGAL_dependencies)
    /usr/local/lib/cmake/CGAL/CGALConfig.cmake:137 (CGAL_setup_target_dependencies)
    CMakeLists.txt:16 (find_package)
  
  
  CMake Error at /usr/local/lib/cmake/CGAL/CGAL_SetupGMP.cmake:69 (target_include_directories):
    Cannot specify include directories for imported target "CGAL".
  Call Stack (most recent call first):
    /usr/local/lib/cmake/CGAL/CGAL_SetupCGALDependencies.cmake:80 (use_CGAL_GMP_support)
    /usr/local/lib/cmake/CGAL/CGAL_setup_target_dependencies.cmake:3 (CGAL_setup_CGAL_dependencies)
    /usr/local/lib/cmake/CGAL/CGALConfig.cmake:137 (CGAL_setup_target_dependencies)
    CMakeLists.txt:16 (find_package)
  
  
  CMake Error at /usr/local/lib/cmake/CGAL/CGAL_SetupGMP.cmake:71 (target_compile_definitions):
    Cannot specify compile definitions for imported target "CGAL".
  Call Stack (most recent call first):
    /usr/local/lib/cmake/CGAL/CGAL_SetupCGALDependencies.cmake:80 (use_CGAL_GMP_support)
    /usr/local/lib/cmake/CGAL/CGAL_setup_target_dependencies.cmake:3 (CGAL_setup_CGAL_dependencies)
    /usr/local/lib/cmake/CGAL/CGALConfig.cmake:137 (CGAL_setup_target_dependencies)
    CMakeLists.txt:16 (find_package)
  
  
  CMake Error at /usr/local/lib/cmake/CGAL/CGAL_SetupCGALDependencies.cmake:94 (target_compile_features):
    Cannot specify compile features for imported target "CGAL".
  Call Stack (most recent call first):
    /usr/local/lib/cmake/CGAL/CGAL_setup_target_dependencies.cmake:3 (CGAL_setup_CGAL_dependencies)
    /usr/local/lib/cmake/CGAL/CGALConfig.cmake:137 (CGAL_setup_target_dependencies)
    CMakeLists.txt:16 (find_package)
  
  
  -- CGAL version: 5.3
  -- Build type: Release
  -- USING CXXFLAGS = '-DVERSION_INFO=\"3.3.0\" -O3 -DNDEBUG'
  -- USING EXEFLAGS = ' '
  -- Found PythonInterp: /usr/local/bin/python (found version "3.6.5")
  -- Found PythonLibs: /usr/local/lib/libpython3.6m.so
  -- Performing Test HAS_FLTO
  -- Performing Test HAS_FLTO - Success
  -- Found pybind11: /usr/local/include (found version "2.6.2" dev1)
  -- Found pybind11 v2.6.2: /usr/local/include;/usr/local/include/python3.6m
  -- Configuring incomplete, errors occurred!
```

**Comment by lrineau** [comment]
Hi Keith, we did change a lot in our CMake scripts, about 4 days ago (PR #5063). Unfortunately, the oldest version of CMake we tested with is the one in Debian Stable: CMake version 3.13.4, and for CGAL-5.3 we plan to document that CGAL requires CMake-3.13 at least. I think the error your CMake-3.7 complains about is something that has only been allowed since CMake version 3.11, see [the changelog](https://cmake.org/cmake/help/latest/release/3.11.html#other):
> Alias Targets may now alias Imported Targets that are created with the GLOBAL option to add_library().

Unfortunately, that means that you will have to upgrade the CMake version on the CI server.
